client/packages/android/app/src/main/java/org/openmsupply/client/NativeApi.java

```diff
@@ -271,4 +286,27 @@ public void onDiscoveryStopped(String serviceType) {
             startServerDiscovery(null);
         }
     }
+
+    public class omSupplyServer {
```

Comment:
Added this helper class since needed to share getUrl() with CertWebViewClient (also added other getters for readability in CertWebViewClient)